src/git/install.sh

```diff
@@ -16,16 +16,48 @@ keyserver hkp://keyserver.ubuntu.com:80
 keyserver hkps://keys.openpgp.org
 keyserver hkp://keyserver.pgp.com"
 
-set -e
-
-# Clean up
-rm -rf /var/lib/apt/lists/*
-
 if [ "$(id -u)" -ne 0 ]; then
     echo -e 'Script must be run as root. Use sudo, su, or add "USER root" to your Dockerfile before running this script.'
     exit 1
 fi
 
+# Bring in ID, ID_LIKE, VERSION_ID, VERSION_CODENAME
+. /etc/os-release
+# Get an adjusted ID independent of distro variants
+if [ "${ID}" = "debian" ] || [ "${ID_LIKE}" = "debian" ]; then
+    ADJUSTED_ID="debian"
+elif [[ "${ID}" = "rhel" || "${ID}" = "fedora" || "${ID}" = "mariner" || "${ID_LIKE}" = *"rhel"* || "${ID_LIKE}" = *"fedora"* || "${ID_LIKE}" = *"mariner"* ]]; then
+    ADJUSTED_ID="rhel"
+    VERSION_CODENAME="${ID}{$VERSION_ID}"
+else
+    echo "Linux distro ${ID} not supported."
+    exit 1
+fi
+
+if type apt-get > /dev/null 2>&1; then
+    INSTALL_CMD=apt-get
+elif type dnf > /dev/null 2>&1; then
+    INSTALL_CMD=dnf
+elif type microdnf > /dev/null 2>&1; then
```

Comment:
With my limited knowledge of Fedora distros, I believe Fedora-based minimal containers can have both `dnf` and `microdnf` package managers present. 

In this case, do we want to give priority to `microdnf` as the user might be looking for slim down containers (ie. should the check for `microdnf` precede the `dnf` check? Curious on what you think 💭

Reply:
Quite happy to reverse the check/priority. I'll do so and run the tests, if they pass expect to see the switch in a commit/updated PR.

Reply:
Commit cba4c71 resolves this request.